dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -89,15 +89,32 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     {
         if (IsPackagedAppScenario())
         {
-            return m_channel.PushNotificationReceived([weak_self = get_weak(), handler](auto&&, auto&& args)
+            if (m_channel)
             {
-                if (auto strong = weak_self.get())
+                return m_channel.PushNotificationReceived([weak_self = get_weak(), handler](auto&&, auto&& args)
+                {
+                    if (auto strong = weak_self.get())
+                    {
+                        auto pushArgs = winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(args);
+                        pushArgs.Handled(true);
+                        handler(*strong, pushArgs);
+                    };
+                });
+            }
+            else
```

Comment:
Comment needed here that this else is specifically for the RemoteId case when we directly call into the UDK from the SDK to requestChannel from RemoteId